src/internal/TransmuteCollection.js

```diff
@@ -121,6 +137,20 @@ export const has = protocol({
   name: 'has',
 });
 
+/**
+ * @private
+ * Return a `Seq.Indexed` of the keys in `subject`
+ * 
+ * @param {TYPE<_, V>} subject
+ * @return {Seq.Indexed<V>}
+ */
+export const indexedSeq = protocol({
```

Comment:
The original implementation used to build sequences based on whether or not the subject was an indexed or a keyed iterable, and reconstruct the object using that new sequence. I figured since the work was already done, there's no reason to toss it out.

Reply:
The idea of `TransmuteCollection` was to have a set of protocols that are used internally by the library, so if you have a custom type in client code you can implement the protocols and that type would work with transmute. I'd lean away from adding an `indexedSeq` protocol until it's used by something in the library.